src/main/java/com/jfrog/ide/idea/scan/data/ScanConfig.java

```diff
@@ -95,6 +95,19 @@ public void setSkippedFolders(List<String> skippedFolders) {
         this.skippedFolders = skippedFolders;
     }
 
+    @Override
+    public String toString() {
+        return "ScanConfig{" +
+                "scanType=" + scanType +
+                ", language='" + language + '\'' +
+                ", roots=" + roots +
+                ", output='" + output + '\'' +
+                ", grepDisable=" + grepDisable +
+                ", cves=" + cves +
+                ", skippedFolders=" + skippedFolders +
+                ", excludedRules=" + excludedRules +
+                '}';
+    }
```

Comment:
You can replace it by a simple `@ToString` Lombok annotation above the class config:
```java
@ToString
public class ScanConfig {
```